hs_discover/src/main.js

```diff
@@ -1,13 +1,13 @@
 import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap-vue/dist/bootstrap-vue.css';
 import BootstrapVue from 'bootstrap-vue';
 import Vue from 'vue';
-// import App from './App.vue';
-// import router from './router';
-import Ping from './components/Ping.vue'
+import Search from './components/Search.vue';
 
-Vue.use(BootstrapVue)
+// Vue.config.devtools = true;
+Vue.use(BootstrapVue);
+Vue.prototype.mapTest = window.mapTest;
```

Comment:
mapTest?  Should this be removed or renamed?  I don't see any references to maptest elsewhere.

Reply:
Commit bfa494506